tests/csapi/room_leave_test.go

```diff
@@ -0,0 +1,262 @@
+package csapi_tests
+
+import (
+	"net/url"
+	"strings"
+	"testing"
+
+	"github.com/tidwall/gjson"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+func TestLeftRoomFixture(t *testing.T) {
+	deployment := Deploy(t, b.MustValidate(b.Blueprint{
```

Comment:
Inline blueprints are expensive to make as they get snapshotted, please don't do this unless you have to. Just use `BlueprintOneToOneRoom` and create charlie via [RegisterUser](https://github.com/matrix-org/complement/blob/main/internal/docker/deployment.go#L84).